conans/client/cmd/export.py

```diff
@@ -89,7 +90,33 @@ def _capture_export_scm_data(conanfile, conanfile_dir, destination_folder, outpu
     # Generate the scm_folder.txt file pointing to the src_path
     src_path = scm.get_repo_root()
     save(scm_src_file, src_path.replace("\\", "/"))
-    scm_data.replace_in_file(os.path.join(destination_folder, "conanfile.py"))
+
```

Comment:
I would feel a bit more comfortable extracting this to a `replace_scm` function. The overall together is a bit scary.